File: Matriz.js
//Matriz Multidimencional

console.log("Ingrese sus siguientes datos")
let matriz = []

//Numero telefonico 

while(true){
    console.log("Para salir ingrese: Terminado")
    let nombre = prompt("Ingrese su nombre:")
    if(nombre == "Terminado") {break};
    let apellido = prompt("Ingrese su apellido:")
    let dni = prompt("Ingrese su dni:")
    //Lista de numero telefonico
    let numero = []
    while(true){
      let numeroTelefonico = prompt("Ingrese su numero telefonico")
      if(numeroTelefonico == "Terminado"){break}
    numero.push(numeroTelefonico)}
    matriz.push([nombre, apellido, dni, numero])
}

console.log(matriz)

File: Matriz.py
#Matriz multidimencional


matriz = []

while True:
   print("Ingrese sus datos(Para salir escriba: 'Terminado'")
   nombre = input("Ingrese su nombre:")
   if nombre == "Terminado":
      print("A salido")
      break
   apellido = input("Ingrese su apellido:")
   dni = input("Ingrese su dni:")
   #Cargas de numero telefonico
   numero = []
   while True:
     print("Para salir escriba:'Terminado'")
     numero_telefono = input("Ingrese su numeor de telefono:")
     if numero_telefono == "Terminado":
        print("A salido")
        break
     #Se agrega numero de telefono a la lista numero
     numero.append(numero_telefono)
   matriz.append([nombre, apellido, dni, numero])
print("\nMatriz final:")
for persona in matriz:
    print(persona)